project-base/build.xml

```diff
@@ -11,4 +11,17 @@
     <property name="phpstan.level" value="4"/>
 
     <import file="${path.framework}/build.xml"/>
+
+    <target name="stylelint" description="Run CSS linter.">
+        <exec executable="${path.stylelint.executable}" logoutput="true" passthru="true" checkreturn="true">
+            <arg path="src/**/*.less"/>
+        </exec>
+    </target>
+
+    <target name="stylelint-fix" description="Run CSS linter.">
+        <exec executable="${path.stylelint.executable}" logoutput="true" passthru="true" checkreturn="true">
+            <arg path="src/**/*.less"/>
```

Comment:
We should have some variable for that `src` as is in framework's build.xml

Reply:
Resolved as the target was removed from project-base and in frameworks build.xml is the variable you mentioning